app/controllers/votes_controller.rb

```diff
@@ -0,0 +1,22 @@
+class VotesController < ApplicationController
+
+  def create
+    @vote = Vote.new(vote_params)
+    @vote.user = current_user
+    if params[:up_vote]
+      @vote.vote_type = 1
+    elsif params[:down_vote]
+      @vote.vote_type = -1
+    end
+    if (@vote.save)
+      redirect_back(fallback_location: root_path)
+    else
+      redirect_back(fallback_location: root_path)
+      # Not sure how to render errors
```

Comment:
Cool. Make a ticket for this and we can work on it next week.

Reply:
#25